src/index.scss

```diff
@@ -1,9 +1,8 @@
-@import "~@edx/edx-bootstrap/scss/edx/theme.scss";
+@import "~@edx/paragon/scss/edx/theme.scss";
+@import "~@edx/paragon/scss/edx/fonts.scss";
 
 $fa-font-path: "~font-awesome/fonts";
-@import "~font-awesome/scss/font-awesome";
 
-@import "~@edx/paragon/src/index.scss";
```

Comment:
3 deletes and 2 adds.  Probably fine, but just checking. :)

Reply:
Wasn't fine ;) It is now

Reply:
So `index.css` went to `fonts.css`?  Also, does order matter, since we are not loading this before `$fa-font-path` setup?

Reply:
the order of `fonts.scss` and `scss/edx/theme.scss` doesn't matter. `index.scss` is now unneeded since edx-bootstrap and paragon are now merged. `$fa-font-path` is an unrelated font-awesome thing.